test/integration/targets/vault/vaulted.inventory.toml

```diff
@@ -0,0 +1,12 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
Store this file in the clear and encrypt it during the test. This will make the test more maintainable.

Reply:
ok, I will do it.

Reply:
done.